Description: fix some spelling errors in upstream files
 Fix spelling errors in source and manpages.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Forwarded: http://bugs.icu-project.org/trac/ticket/13436
Last-Update: 2017-10-19

---

--- icu-59.1.orig/source/extra/uconv/uconv.1.in
+++ icu-59.1/source/extra/uconv/uconv.1.in
@@ -109,7 +109,7 @@ When calling
 it is possible to specify callbacks that are used to handle invalid
 characters in the input, or characters that cannot be transcoded to
 the destination encoding. Some encodings, for example, offer a default
-substitution character that can be used to represent the occurence of
+substitution character that can be used to represent the occurrence of
 such characters in the input. Other callbacks offer a useful visual
 representation of the invalid data.
 .PP
@@ -418,10 +418,10 @@ and map Katakana to Hiragana:
 .B "      \-x '::nfkc; [:Cc:] >; ::katakana-hiragana;'"
 .SH CAVEATS AND BUGS
 .B uconv
-does report errors as occuring at the first invalid byte
+does report errors as occurring at the first invalid byte
 encountered. This may be confusing to users of GNU
 .BR iconv (1),
-which reports errors as occuring at the first byte of an invalid
+which reports errors as occurring at the first byte of an invalid
 sequence. For multi-byte character sets or encodings, this means that
 .BR uconv
 error positions may be at a later offset in the input stream than
@@ -432,7 +432,7 @@ The reporting of error positions when a
 inaccurate or unavailable, in which case
 .BR uconv
 will report the offset in the output stream at which the error
-occured.
+occurred.
 .SH AUTHORS
 Jonas Utterstroem
 .br
--- icu-59.1.orig/source/test/intltest/incaltst.cpp
+++ icu-59.1/source/test/intltest/incaltst.cpp
@@ -443,7 +443,7 @@ void IntlCalendarTest::TestBuddhistForma
     }
     delete fmt2;
     
-    CHECK(status, "Error occured testing Buddhist Calendar in English ");
+    CHECK(status, "Error occurred testing Buddhist Calendar in English ");
     
     status = U_ZERO_ERROR;
     // Now, try in Thai
@@ -563,7 +563,7 @@ void IntlCalendarTest::TestJapaneseForma
 
     delete cal2;
     delete fmt2;
-    CHECK(status, "Error occured");
+    CHECK(status, "Error occurred");
     
     // Now, try in Japanese
     {
@@ -875,7 +875,7 @@ void IntlCalendarTest::TestPersianFormat
     }
     delete fmt2;
     
-    CHECK(status, "Error occured testing Persian Calendar in English "); 
+    CHECK(status, "Error occurred testing Persian Calendar in English "); 
 }
 
 
@@ -895,7 +895,7 @@ void IntlCalendarTest::simpleTest(const
         }
 
         d = fmt2->parse(expect,status);
-        CHECK(status, "Error occured parsing " + UnicodeString(loc.getName()));
+        CHECK(status, "Error occurred parsing " + UnicodeString(loc.getName()));
         if(d != expectDate) {
             fmt2->format(d,tmp);
             errln(UnicodeString("Failed to parse " ) + escape(expect) + ", " + loc.getName() + " expect " + (double)expectDate + " got " + (double)d  + " " + escape(tmp));
--- icu-59.1.orig/source/test/intltest/intltest.cpp
+++ icu-59.1/source/test/intltest/intltest.cpp
@@ -707,7 +707,7 @@ void IntlTest::runIndexedTest( int32_t /
         default: name = ""; break;
     }
     */
-    this->errln("*** runIndexedTest needs to be overriden! ***");
+    this->errln("*** runIndexedTest needs to be overridden! ***");
 }
 
 
--- icu-59.1.orig/source/tools/ctestfw/uperf.cpp
+++ icu-59.1/source/tools/ctestfw/uperf.cpp
@@ -334,7 +334,7 @@ UPerfFunction* UPerfTest::runIndexedTest
         default: name = ""; break;
     }
     */
-    fprintf(stderr,"*** runIndexedTest needs to be overriden! ***");
+    fprintf(stderr,"*** runIndexedTest needs to be overridden! ***");
     return NULL;
 }
 
--- icu-59.1.orig/source/tools/genrb/genrb.cpp
+++ icu-59.1/source/tools/genrb/genrb.cpp
@@ -652,7 +652,7 @@ processFile(const char *filename, const
         goto finish;
     }
     if (ucbuf == NULL || U_FAILURE(status)) {
-        fprintf(stderr, "An error occured processing file %s. Error: %s\n",
+        fprintf(stderr, "An error occurred processing file %s. Error: %s\n",
                 openFileName == NULL ? filename : openFileName, u_errorName(status));
         goto finish;
     }
--- icu-59.1.orig/source/tools/genrb/parse.cpp
+++ icu-59.1/source/tools/genrb/parse.cpp
@@ -362,7 +362,7 @@ parseUCARules(ParseState* state, char *t
     ucbuf = ucbuf_open(filename, &cp, getShowWarning(),FALSE, status);
 
     if (U_FAILURE(*status)) {
-        error(line, "An error occured while opening the input file %s\n", filename);
+        error(line, "An error occurred while opening the input file %s\n", filename);
         return NULL;
     }
 
@@ -500,7 +500,7 @@ parseTransliterator(ParseState* state, c
     ucbuf = ucbuf_open(filename, &cp, getShowWarning(),FALSE, status);
 
     if (U_FAILURE(*status)) {
-        error(line, "An error occured while opening the input file %s\n", filename);
+        error(line, "An error occurred while opening the input file %s\n", filename);
         return NULL;
     }
 
@@ -758,7 +758,7 @@ GenrbImporter::getRules(
         return;
     }
     if (ucbuf.isNull() || U_FAILURE(errorCode)) {
-        fprintf(stderr, "An error occured processing file %s. Error: %s\n", openFileName.data(), u_errorName(errorCode));
+        fprintf(stderr, "An error occurred processing file %s. Error: %s\n", openFileName.data(), u_errorName(errorCode));
         return;
     }
 
--- icu-59.1.orig/source/tools/gensprep/gensprep.c
+++ icu-59.1/source/tools/gensprep/gensprep.c
@@ -113,7 +113,7 @@ static int printHelp(int argc, char* arg
     fprintf(stderr,
         "\t-d or --destdir          destination directory, followed by the path\n"
         "\t-s or --sourcedir        source directory of ICU data, followed by the path\n"
-        "\t-b or --bundle-name      generate the ouput data file with the name specified\n"
+        "\t-b or --bundle-name      generate the output data file with the name specified\n"
         "\t-i or --icudatadir       directory for locating any needed intermediate data files,\n"
         "\t                         followed by path, defaults to %s\n",
         u_getDataDirectory());
--- icu-59.1.orig/source/tools/makeconv/makeconv.1.in
+++ icu-59.1/source/tools/makeconv/makeconv.1.in
@@ -51,14 +51,14 @@ be understood by
 .BR makeconv .
 The ICU ucm format is similar to the IBM NLTC upmap/tpmap/rpmap files.
 Comments in the
-.I convertable
+.I convertible
 are handled as follows. If a comment (starting with a `#' sign) that
 is after some text does contain the fallback indicator `|' then only
 the text starting with the `#' sign, and ending before the `|' sign,
 is ignored.
 Otherwise, or if the comment is the first thing on the line,
 the comment runs up to the end of the line. This special
-handling of comments is to accomodate the practice of putting fallback
+handling of comments is to accommodate the practice of putting fallback
 information in comments in the strict IBM NLTC ucmap format.
 .PP
 Note that new converters will be automatically found by ICU after their
